docker-entrypoint.sh

```diff
@@ -12,11 +12,11 @@
 
 set -eu
 
-raw_cloud_config_dir="/usr/code/cloudconfig/openstack/latest"
-raw_cloud_config_path="$raw_cloud_config_dir/user_data"
+RAW_CLOUD_CONFIG_DIR="/usr/code/cloudconfig/openstack/latest"
+RAW_CLOUD_CONFIG_PATH="${RAW_CLOUD_CONFIG_DIR}/user_data"
```

Comment:
This is just for my sanity. Made bash vars upper case and surrounded by curly braces.